src/turbine/runtime/platform.py

```diff
@@ -189,9 +193,11 @@ async def write(
             else:
                 print(f"Successfully created {resp[1].name} connector")
 
-            print(f"Creating application: {self.app_config.name}")
-            await self._create_application(self._pipelineName)
-            print(f"Successfully created application: {self.app_config.name}")
+            if not globals().get("__application_id"):
```

Comment:
technically, it's a uuid.. update is for here and elsewhere
```suggestion
            if not globals().get("__application_uuid"):
```